scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl

```diff
@@ -59,12 +54,12 @@ workflow CNVSomaticPanelWorkflow {
         }
     }
 
-    scatter (normal_bam in normal_bams) {
+    scatter (i in range(length(normal_bams))) {
```

Comment:
Is there any way to verify that `length(normal_bams) == length(normal_bais)`?  And is this the sort of thing we should be doing in WDLs?

Reply:
Looking into how we would do that.

I'd do checks like this to a minimal degree.  I can make a change that will cause it to fail when dealing with pairs.

Reply:
OK.  If it's not something we're going to do by convention then I wouldn't worry about it.

Reply:
I changed the bam and bais to a pair.  That should fail right away if the counts are off (I believe).

Done